contrib/docker/.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+.dockerignore
+README.txt
+.git
```

Comment:
Dockerfile and .dockerignore are too small to matter for ignoring purposes imo. Aint .dockerignore itself autoignored?
README.txt is also small though.
My recommendation is to only leave .git.

Reply:
Size does not really matter in this case. It is about cache busting. If I change the README the Docker Image does not need to rebuild.